internal/interceptor/server.go

```diff
@@ -270,7 +271,10 @@ func (s *Server) HandleRoot(w http.ResponseWriter, r *http.Request) {
 
 	pData.Environment = selectEnvironmentFromMapping(odsConfig.BranchToEnvironmentMapping, pData.GitRef)
 
-	rendered, err := renderPipeline(odsConfig, pData)
+	// Enable it once OpenShift Pipelines 1.5 is available in the cluster
+	// https://docs.openshift.com/container-platform/4.8/cicd/pipelines/op-release-notes.html#op-release-notes-1-5_op-release-notes
+	enableAggregateTaskStatus := false
```

Comment:
@michaelsauter To avoid breaking until OpenShift Pipelines 1.5 is available, we have a control variable that is disabled by default.

Reply:
Let's keep it in draft like this for now. I would like to remove the flag altogether from the code base and make 1.5 the minimum requirement for ods-pipeline v0.1.0. Without build status, the pipeline does not make sense for me.